src/components/switch/Switch.constants.ts

```diff
@@ -0,0 +1,25 @@
+import {
+    TSwitchSize,
+    TSwitchState,
+    TSwitchWidths,
+    TSwitchHeights,
+    TSwitchInnerWidths,
+} from './Switch.types';
+
+const DEFAULT_SWITCH_SIZE: TSwitchSize = 'md';
+const DEFAULT_SWITCH_STATE: TSwitchState = 'on';
```

Comment:
IMO it would be better to have a `checked`, `selected`, `on` or `ticked` prop and a separate `disabled` prop to be consistent among other input components. It would also remove some complexity in typing and usage.